lib/components/Pagination.js

```diff
@@ -2,42 +2,24 @@ import React from "react";
 import Dropdown from "./Dropdown";
 import classnames from "classnames";
 
-const LeftSVG = () => (
-  <svg className="w-5 h-5" viewBox="0 0 20 20" fill="currentColor">
-    <path
-      fillRule="evenodd"
-      d="M12.707 5.293a1 1 0 010 1.414L9.414 10l3.293 3.293a1 1 0 01-1.414 1.414l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 0z"
-      clipRule="evenodd"
-    />
-  </svg>
-);
+const Pagination = ({ count = 0, pageNo, pageSize, navigate, className = "", emptyPageMsg = "No records" }) => {
+  const maxPageNo = Math.ceil(count / pageSize);
+  const isPageEmpty = count === 0;
+  const startIndexOfCurrentPage = isPageEmpty ? 0 : ((pageNo - 1) * pageSize) + 1;
+  const endIndexOfCurrentPage = Math.min(count, pageNo * pageSize);
```

Comment:
Is this logic really needed? In what case will count be greater than `pageNo * pageSize`?